src/provider/element-templates/ElementTemplates.js

```diff
@@ -194,8 +206,41 @@ export default class ElementTemplates {
 
     this._commandStack.execute('propertiesPanel.camunda.changeTemplate', context);
 
+    this._eventBus.fire(`elementTemplates.${action}`, { element, newTemplate });
+
     return context.element;
   }
+
+  /**
+   * Remove template from a given element.
+   *
+   * @param {djs.model.Base} element
+   */
+  removeTemplate(element) {
+    removeTemplateHelper(element, this._injector);
```

Comment:
Any reason we don't inline the `removeTemplateHelper` here? Any reason we don't emit `elementTemplate.remove`?

Reply:
The reason is that this is the same logic for platform and cloud. Previously we were using re-exporting this utility in the cloud utilities file. This is so we can keep re-using the same code - easier maintenance.

The event is within the utility too.

Reply:
Re-use in class hierarchies is as simple as not implementing the method (in Cloud-element templates).

So I believe we could get rid of the helper entirely, and use inheritance.

Reply:
Of course, functional composition works, too.